engine/src/main/java/com/datatorrent/stram/client/StramAppLauncher.java

```diff
@@ -630,6 +631,12 @@ public ApplicationId launchApp(AppFactory appConfig) throws Exception
       client.setArchives(conf.get(ARCHIVES_CONF_KEY_NAME));
       client.setOriginalAppId(conf.get(ORIGINAL_APP_ID));
       client.setQueueName(conf.get(QUEUE_NAME));
+      String tags = conf.get(TAGS);
+      if (tags != null) {
+        for (String tag : tags.split(",")) {
+          client.addTag(tag.trim());
```

Comment:
That's not necessary. The _apex. prefix added to this PR is on the property names, not the tag values